NCS.DSS.Outcomes/DeleteOutcomesHttpTrigger/Function/DeleteOutcomesHttpTrigger.cs

```diff
@@ -1,75 +1,80 @@
         {
             log.LogInformation("Delete Action Plan C# HTTP trigger function processed a request.");
 
             if (!Guid.TryParse(customerId, out var customerGuid))
-                return httpResponseMessageHelper.BadRequest(customerGuid);
+                return new BadRequestObjectResult(customerGuid);
 
             if (!Guid.TryParse(interactionId, out var interactionGuid))
-                return httpResponseMessageHelper.BadRequest(interactionGuid);
+                return new BadRequestObjectResult(interactionGuid);
 
             if (!Guid.TryParse(actionplanId, out var actionplanGuid))
-                return httpResponseMessageHelper.BadRequest(actionplanGuid);
+                return new BadRequestObjectResult(actionplanGuid);
 
             if (!Guid.TryParse(outcomeId, out var outcomesGuid))
-                return httpResponseMessageHelper.BadRequest(outcomesGuid);
+                return new BadRequestObjectResult(outcomesGuid);
 
-            var doesCustomerExist = await resourceHelper.DoesCustomerExist(customerGuid);
+            var doesCustomerExist = await _resourceHelper.DoesCustomerExist(customerGuid);
 
             if (!doesCustomerExist)
-                return httpResponseMessageHelper.NoContent(customerGuid);
+                return new NoContentResult();
 
-            var doesActionPlanExist = resourceHelper.DoesActionPlanResourceExistAndBelongToCustomer(actionplanGuid, interactionGuid, customerGuid);
+            var doesActionPlanExist = _resourceHelper.DoesActionPlanResourceExistAndBelongToCustomer(actionplanGuid, interactionGuid, customerGuid);
 
             if (!doesActionPlanExist)
-                return httpResponseMessageHelper.NoContent(actionplanGuid);
+                return new NoContentResult();
 
-            var outcome = await outcomesDeleteService.GetOutcomeForCustomerAsync(customerGuid, interactionGuid, actionplanGuid, outcomesGuid);
+            var outcome = await _outcomesDeleteService.GetOutcomeForCustomerAsync(customerGuid, interactionGuid, actionplanGuid, outcomesGuid);
 
             if (outcome == null)
-                return httpResponseMessageHelper.NoContent(outcomesGuid);
-
-            var outcomeDeleted = await outcomesDeleteService.DeleteAsync(outcome.OutcomeId.GetValueOrDefault());
+                return new NoContentResult();
 
-            return !outcomeDeleted ?
-                httpResponseMessageHelper.BadRequest(outcomesGuid) :
-                httpResponseMessageHelper.Ok();
+            var outcomeDeleted = await _outcomesDeleteService.DeleteAsync(outcome.OutcomeId.GetValueOrDefault());
 
+            return !outcomeDeleted
+                ? new BadRequestObjectResult(outcomesGuid)
+                : new JsonResult(outcomesGuid, new JsonSerializerOptions())
+                {
+                    StatusCode = (int)HttpStatusCode.OK
+                };
         }
     }
```

Comment:
Comment out disabled function and add to relevant Confluence page

Reply:
Resolving this comment as it's desired but not required - please make a note for future improvement